Fix timer round-up behaviour

harenbrs opened the pull request:
The timer feature (#99) introduced a bug in the formatting of fractional seconds when the fractional part is >= 0.995:

```python
>>> with yaspin(timer=True) as sp:
...     time.sleep(0.995)
...     sp.ok()
OK  (0:00:00.100)
```

This PR fixes it, making the rounding up work correctly:

```python
>>> with yaspin(timer=True) as sp:
...     time.sleep(0.995)
...     sp.ok()
OK  (0:00:01.00)
```

harenbrs commented:
I could add a test to avoid the drop in code coverage, such as:

```python
def test_timer_rounding():
    sp = yaspin(timer=True)
    sp.start()
    time.sleep(0.995)
    sp.stop()
    sp._freeze("")
    assert "(0:00:01.00)" in sp._last_frame
```

However, this depends on the overhead between `sp.start()` and `sp.stop()` taking at most 5ms. I see potential for flakyness if the machine that's running the test is highly loaded, something I was careful to avoid with the other timer tests.

What do you reckon, @pavdmyt?

pavdmyt reviewed (changes requested):
Hi @harenbrs 

Thanks for the fix! 

Let's add some tests to ensure that rounding logic works as expected. Also, please squash your current commits into 1, to avoid cluttering commit history.

harenbrs commented:
Hi @pavdmyt, squashed and added two tests. I managed to get around the potential test flakiness by setting the `_stop_time` property deterministically.

Btw, I don't know if you requested any changes inline — my force push apparently makes it impossible to see them now.

pavdmyt commented:
Included in [v1.4.1](https://github.com/pavdmyt/yaspin/releases/tag/v1.4.1)
